app/templates/frameworks/services.html

```diff
@@ -65,91 +64,115 @@ <h2 class="summary-item-heading">{{ framework.name }} is closed for applications
-    field_headings=[
-        "Service name",
-        "Progress",
-        "Make a copy"
-    ],
-    field_headings_visible=False
-  ) %}
-    {% call summary.row() %}
-      {{ summary.service_link(draft.serviceName,
-                              url_for(".view_service_submission", framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft.id)) }}
+  {% set ns = namespace(draft_rows = []) %}
+  {% for draft in drafts %}
+    {% set service_link %}
+    <a class="govuk-link" href="{{ url_for('.view_service_submission', framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft.id) }}">{{ draft.serviceName }}</a>
+    {% endset %}
+    {% set make_copy_button %}
+    <form method="post" action="{{ url_for('.copy_draft_service', framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft.id) }}">
+      <input type="hidden" name="csrf_token" value="{{ csrf_token_value or csrf_token() }}" />
+    {{ govukButton({
+      "text": "Make a copy",
+      "classes": "govuk-!-margin-0",
+    }) }}
+    </form>
+    {% endset %}
 
-      {% if framework.status == 'open' %}
-        {{ summary.text(submission.multiline_string(
-          submission.can_be_completed_text(draft.unanswered_required, framework.status),
+    {% set row = ns.draft_rows.append(
+      [
+          {"html": service_link},
+          {"html": submission.multiline_string(submission.can_be_completed_text(draft.unanswered_required, framework.status),
           submission.unanswered_required_text(draft.unanswered_required, draft.unanswered_optional),
-          submission.unanswered_optional_text(draft.unanswered_required, draft.unanswered_optional)
-        )) }}
-        {% call summary.field(action=True) %}
-          <form method="post" action="{{ url_for('.copy_draft_service', framework_slug=framework.slug, lot_slug=draft.lot, service_id=draft.id) }}">
-            <input type="hidden" name="csrf_token" value="{{ csrf_token_value or csrf_token() }}" />
-            {{ govukButton({
-              "text": "Make a copy",
-              "classes": "govuk-!-margin-0",
-            }) }}
-          </form>
-        {% endcall %}
-      {% else %}
-        {{ summary.text() }}
-        {{ summary.text() }}
-      {% endif %}
-    {% endcall %}
-  {% endcall %}
+          submission.unanswered_optional_text(draft.unanswered_required, draft.unanswered_optional))},
+          {"html": make_copy_button},
+      ]
+    ) %}
+{% endfor %}
+<p class="summary-item-top-level-action">
+  <a href="{{ url_for(".start_new_draft_service", framework_slug=framework.slug, lot_slug=lot.slug) }}" class="summary-change-link">Add a service</a>
+</p>
 
+    {% if not drafts %}
+      {{"You haven’t added any services yet." if framework.status == 'open' else "You didn’t add any services."}}
+    {% endif %}
+    {{ govukTable({
+      'head': [
+        {'text': "Service name"},
+        {'text': "Progress"},
+        {'text': "Make a copy"},
```

Comment:
These used to be visually hidden, not sure if we want to replicate that?

Reply:
## Before 
![Screenshot 2021-06-17 at 15 32 39](https://user-images.githubusercontent.com/1764158/122418145-ebe02c00-cf81-11eb-8e29-680ad83d5b62.png)
## After
![Screenshot 2021-06-17 at 15 38 38](https://user-images.githubusercontent.com/1764158/122418574-3497e500-cf82-11eb-85c6-a80c4219b037.png)

Reply:
The last time we had a table like this, we hid "Make a copy", since it's repeated by all the buttons: https://github.com/alphagov/digitalmarketplace-briefs-frontend/blob/afedd090199322b0dfe1545b824e10e03ae7899e/app/templates/buyers/dashboard.html#L41-L44